include/udx.h

```diff
@@ -193,6 +193,8 @@ struct udx_stream {
 
   udx_cirbuf_t outgoing;
   udx_cirbuf_t incoming;
+
+  udx_fifo_t send_queue_other;
```

Comment:
What's the reason for the `_other` postfix?

Reply:
trying to indicate that this is only for outgoing outside the cirbuf, but yea, didn't like it either. ideas?

Reply:
unordered mb?

Reply:
Unordered seems more descriptive 👏🏼

Reply:
will fix

Reply:
now called `unordered`